tutorials/animation/2d_skeletons.rst

```diff
@@ -3,233 +3,260 @@
+   systems. This can be useful for magic effects, jetpacks, etc.
+-  **Custom Colliders**: Set colliders and influence areas in different
+   parts of the skeletons, great for bosses and fighting games.
+-  **Animation Tree**: Allows complex combinations and blending between
+   several animations, the same way it works in 3D.
 
-.. seealso::
+And much more!
 
-   Before starting, we recommend you to go through the
-   :ref:`doc_cutout_animation` tutorial to gain a general understanding of
-   animating within Godot.
+Making of GBot
+~~~~~~~~~~~~~~
 
-For this tutorial, we will be using a single image to construct our character.
-Download it from :download:`gBot_pieces.png <img/gBot_pieces.png>` or save the
-image below.
+For this tutorial, we will use as demo content the pieces of the
+`GBot <https://www.youtube.com/watch?v=S13FrWuBMx4&list=UUckpus81gNin1aV8WSffRKw>`__
+character, created by Andreas Esau.
 
-.. image:: img/gBot_pieces.png
+.. image:: img/tuto_cutout_walk.gif
 
-It is also advised to download the final character image
-:download:`gBot_complete.png <img/gBot_complete.png>` to have a good reference
-for putting the different pieces together.
+Assets can be downloaded here: :download:`gbot_resources.zip <files/gbot_resources.zip>`.
 
-.. image:: img/gBot_complete.png
+Setting up the rig
+~~~~~~~~~~~~~~~~~~
 
-Creating the polygons
----------------------
+Create an empty Node2D as the root of the scene, we will work under it:
 
-Create a new scene for your model (if it's going to be an animated character,
-you may want to use a ``CharacterBody2D``). For ease of use, an empty 2D node is
-created as a root for the polygons.
+.. image:: img/tuto_cutout1.png
 
-Begin with a ``Polygon2D`` node. There is no need to place it anywhere in the
-scene for now, so simply create it like this:
+Create a :ref:`Skeleton2D <class_Skeleton2D>` node as the first child.
+Then create a :ref:`Bone2D <class_Bone2D>` as a child of that node.
+Create a :ref:`Sprite2D <class_Sprite2D>` as a child of the Bone2D and
+add the hip sprite as its texture. Generally, both in 2D and 3D, the hip
+is the root of the skeleton. This makes it easier to animate.
 
-.. image:: img/skel2d1.png
+.. image:: img/tuto_cutout2.png
 
-Select it and assign the texture with the character pieces you have downloaded
-before:
+Next will be the body. The body needs to be a child of the hip bone, so
+create a Bone2D child of the hip bone and a Sprite2D child of the Bone2D for
+the body.
 
-.. image:: img/skel2d2.png
+.. image:: img/tuto_cutout3.png
 
-Drawing a polygon directly is not advised. Instead, open the "UV" dialog for the
-polygon:
+This looks good. Let's see if our hierarchy works as a skeleton by
+rotating the body bone. We can do this be pressing :kbd:`E` to enter rotate mode,
+and dragging with the left mouse button. To exit rotate mode hit :kbd:`ESC`.
 
-.. image:: img/skel2d3.png
+.. image:: img/tutovec_torso1.gif
```

Comment:
The image here shows the body bone with a Y position transform not mentioned in the text